pkg/gadget-context/run.go

```diff
@@ -94,6 +95,19 @@ func (c *GadgetContext) initAndPrepareOperators(paramValues api.ParamValues) ([]
 		}
 	}
 
+	// set defaulfs for params according to gadget's config
+	if cfg, ok := c.GetVar("config"); ok {
+		if v, ok := cfg.(*viper.Viper); ok {
+			defaults := v.GetStringMapString("paramDefaults")
+			for _, p := range params {
+				fullName := p.Prefix + p.Key
+				if val, ok := defaults[fullName]; ok {
+					p.DefaultValue = val
```

Comment:
Should we validate the val matches the `TypeHint` (if it was defined) or we want to leave that validation to each operator? I think it doesn't hurt to do it here.